test/e2e/pages/newFolderFlow.ts

```diff
@@ -32,7 +32,10 @@ export class NewFolderFlow {
 
 		await this.setFolderTemplate(options.folderTemplate);
 		await this.setFolderNameLocation(options);
-		await this.setConfiguration(options);
+
+		if (options.folderTemplate !== FolderTemplate.EMPTY_PROJECT) {
+			await this.setConfiguration(options);
+		}
```

Comment:
A couple additions in this file to handle the Empty Project flow, which does not have a configuration step, and thus has a <kbd>Create</kbd> button at the folder name/location step instead of a Next button.